client/src/Month.jsx

```diff
@@ -21,8 +21,8 @@ class Month extends React.Component {
               <td>Fr</td>
               <td>Sa</td>
             </tr>
-            {this.props.weeks.map((arr) => {
-              return (<Week maxSelectableDate = {this.props.maxSelectableDate} selectedCheckoutOnlyDate = {this.props.selectedCheckoutOnlyDate} dates = {this.props.dates} checkInDate = {this.props.checkInDate} checkOutDate = {this.props.checkOutDate} hoveredDate = {this.props.hoveredDate} changedHoveredDate = {this.props.changedHoveredDate} thisWeek = {arr} dateClicked = {this.props.dateClicked}/>);
+            {this.props.weeks.map((arr, index) => {
+              return (<Week key={index} maxSelectableDate = {this.props.maxSelectableDate} selectedCheckoutOnlyDate = {this.props.selectedCheckoutOnlyDate} dates = {this.props.dates} checkInDate = {this.props.checkInDate} checkOutDate = {this.props.checkOutDate} hoveredDate = {this.props.hoveredDate} changedHoveredDate = {this.props.changedHoveredDate} thisWeek = {arr} dateClicked = {this.props.dateClicked}/>);
```

Comment:
Same thoughts around multi-line. There's a lot of side-scrolling here.

Reply:
yep good call